style.css

```diff
@@ -304,42 +366,48 @@ input[type=submit]:hover {
   transform-origin: bottom left;
 }
 
+/*style for images in main content on homepage starts*/
 .container {
   position: relative;
   display: inline-block;
   text-align: center;
-  border-radius: 50%;
+  border-radius: 6px;
+  background-color: white;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
#added a box shadow for my homepage images and decreased the border radius to only be slightly rounded and used the opacity technique from trial 1 with slight less transparency